src/app/pages/OriginsLaunchpad/pages/SalesDay/index.module.scss

```diff
@@ -1,15 +1,13 @@
-import styled from 'styled-components';
-
-export const TitleImage = styled.img`
+.titleImage {
   width: 55px;
   height: 54px;
-`;
+}
 
-export const TitleContent = styled.div`
+.titleContent {
   font-size: 2.5rem;
```

Comment:
We have a rule that we try to use `@apply` to as many properties as possible and we write new ones only if they don't have the value we need in Tailwind.

So in this case it would like this: 
`@apply tw-font-semibold tw-leading-none tw-tracking-normal tw-ml-8 tw-uppercase;
font-size: 2.5rem;`

As you can see, only `font-size` has a custom value (and sometimes you find out there is no custom value in your styles and you can delete a selector). Please, go through all of your `index.module.scss` in this PR and adjust them.

Reply:
I updated the styling as you specified.

Reply:
It looks good now, I just found 3 selectors that can be deleted (see my latest comments).